packages/compile-solidity/index.js

```diff
@@ -74,6 +46,10 @@ const Compile = {
 
   // this takes an array of paths and options
   async sourcesWithDependencies({ paths, options }) {
+    if (options.compilers.solc.version === "analyzePragmas") {
```

Comment:
What do you think of shortening this to just `version: "pragma"`?

Reply:
I have no big problem with it, what about you @davidmurdoch?

Reply:
updated